CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
     * Avoid allocations in AbstractSyntaxIndex<>.GetIndexAsync( PR: [#74075](https://github.com/dotnet/roslyn/pull/74075))
 * Bump xamltools to 17.12.35103.251 (PR: [#7309](https://github.com/dotnet/vscode-csharp/pull/7309))
   * Fixed issue with Exception type related to https://github.com/microsoft/vscode-dotnettools/issues/1247
+* Renamed settings (PR: [#7356](https://github.com/dotnet/vscode-csharp/pull/7356))
```

Comment:
Your branch is out of date - this should be moved to the new Latest section in the up to date main